modules/auxiliary/admin/edirectory/edirectory_dhost_cookie.rb

```diff
@@ -22,7 +22,6 @@ def initialize(info = {})
       },
       'References'     =>
```

Comment:
this one for example has no references after deleting the osvdb entry. Should we remove the whole References hash or is at least one reference required?